pkg/controllers/user/networkpolicy/netpol.go

```diff
@@ -91,6 +91,9 @@ func (npmgr *netpolMgr) programNetworkPolicy(projectID string) error {
 	logrus.Debugf("netpolMgr: programNetworkPolicy: namespaces=%+v", namespaces)
 
 	for _, aNS := range namespaces {
+		if IsSystemNamespace(aNS.Name) {
```

Comment:
being that IsSystemNamespace builds a map it seems overly expensive to do on each loop.  can you get the map once (func GetSystemNamespace() map[string]bool) and then check the map in the loops?